src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSource.java

```diff
@@ -134,6 +134,8 @@
     private static final Logger LOGGER = Logger.getLogger(BitbucketSCMSource.class.getName());
     private static final String CLOUD_REPO_TEMPLATE = "{/owner,repo}";
     private static final String SERVER_REPO_TEMPLATE = "/projects{/owner}/repos{/repo}";
+    // TODO: 5 is just a testing value, if we are keeping this, set default value to 0
```

Comment:
What is the reason for leaving this `TODO` and not actually setting the default to `0`?

Reply:
I'm not sure. My vote would be to match the configuration used for the same purpose in [`GitHubSCMSource`](https://github.com/jenkinsci/github-branch-source-plugin/blob/73866ba7a90363f8dcc699e122a8ca2b5da30d31/src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java#L163) unless there is a particular reason Bitbucket should be different.

Reply:
This PR was created for a specific user to test in their environment. If it had worked, I would have set the default to 0 to match `GitHubSCMSource` and taken this PR out of draft. The input from the user was that this did not work, so I had forgotten to close this PR.

Reply:
> The input from the user was that this did not work

Note though that the user did not successfully increase the configured delay to see if that made a difference. They used `System.setProperty` but the property is only read into the variable once at startup, so you either have to change Jenkins' startup config or use Groovy to modify the variable directly.

Also just to clarify, right now [`GitHubSCMSource`](https://github.com/jenkinsci/github-branch-source-plugin/blob/73866ba7a90363f8dcc699e122a8ca2b5da30d31/src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java#L163-L164) uses `5` as the default value.

Reply:
Ah, you're totally right. I remember now I set it to 5 to test because GitHubSCMSource had 5 as default, but wanted this PR default to be 0 if we merged this in so as to not change expected behavior. Makes sense to match GitHubSCMSource.

Reply:
> This PR was created for a specific user to test in their environment. If it had worked, I would have set the default to 0 to match GitHubSCMSource and taken this PR out of draft. The input from the user was that this did not work, so I had forgotten to close this PR.

@car-roll so this is not currently working, and should be closed?

Reply:
@artheus 
Sounds like that is that case.  Is there a particular reason you're concerned about this PR?  Are you volunteering to start reviewing issues and grooming PRs?  That would be awesome!